lib/yardstick/decorator.rb

```diff
@@ -0,0 +1,60 @@
+module Yardstick
+  # A string decorator for applying unix console codes
+  class Decorator
+    FORMAT = "\e[%{mode};%{color}%{string}\e[0m".freeze
+
+    COLOR_CODES = {
+      red:    31,
+      yellow: 33
+    }.freeze
+
+    MODE_CODES = {
+      bold:      1,
+      underline: 4
+    }.freeze
+
+    private_constant(:FORMAT, :COLOR_CODES, :MODE_CODES)
```

Comment:
Lets go for the most clean code, and report the YARD issue upstream. I prefer clean code over working around buggy code. The pain should be visible (Also YARD should crash and not "silence").